packages/shared/src/components/profile/ProfileForm.tsx

```diff
@@ -53,11 +53,12 @@ export default function ProfileForm({
   const { user, updateUser } = useContext(AuthContext);
   const formRef = useRef<HTMLFormElement>(null);
 
-  const emptyTimezoneMode =
-    mode === 'update' ? 'Europe/London' : getUserDefaultTimezone();
-  const initialTimezone = user?.timezone ? user?.timezone : emptyTimezoneMode;
-
-  const [userTimeZone, setUserTimeZone] = useState<string>(initialTimezone);
+  const [userTimeZone, setUserTimeZone] = useState<string>(
+    getUserInitialTimezone({
+      userTimezone: user?.timezone,
+      update: !!(mode === 'update'),
```

Comment:
`!!` is not needed as it's already a boolean

```suggestion
      update: mode === 'update',
```